oxd-server/src/main/java/org/gluu/oxd/server/persistence/SqlPersistenceServiceImpl.java

```diff
@@ -341,4 +345,14 @@ public boolean deleteAllExpiredObjects() {
             IOUtils.closeSilently(conn);
         }
     }
+
+    private static String getKeyColumnName(ConfigurationService configurationService) {
+        String driverClass = configurationService.getConfiguration().getStorageConfiguration().get("driver")!= null ?
+                configurationService.getConfiguration().getStorageConfiguration().get("driver").asText() : null;
+
+        if (!Strings.isNullOrEmpty(driverClass) && driverClass.contains("mysql")) {
+            return "`key`";
```

Comment:
if it's `MySQL` you meant to return `obj_key`, no ? (if `key` is not allowed)

Reply:
@yuriyz , I found Mysql is allowing `key` column name if it is enclosed by ` character. But other databases (h2, postgres) donot allow ` character in query. So to keep the column name `key` for all databases we are using ` in Mysql database.